pkg/acsengine/engine.go

```diff
@@ -136,6 +137,7 @@ var kubernetesAddonYamls15 = map[string]string{
 	"MASTER_ADDON_KUBERNETES_DASHBOARD_DEPLOYMENT_B64_GZIP_STR": "kubernetesmasteraddons-kubernetes-dashboard-deployment1.5.yaml",
 	masterAddonAzureStorageClasses:                              masterAddonAzureStorageClassesFileUnmanaged,
 	"MASTER_ADDON_TILLER_DEPLOYMENT_B64_GZIP_STR":               "kubernetesmasteraddons-tiller-deployment1.5.yaml",
+	"MASTER_ADDON_ACI_CONNECTOR_DEPLOYMENT_B64_GZIP_STR":        "kubernetesmasteraddons-aci-connector-deployment1.5.yaml",
```

Comment:
Do we really need this separate file?  I don't think we are supporting K8s 1.5 and your yaml is the same for the 1.5 and the non-1.5 version, yes?